packages/tabs/src/TabTitle.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useRef, useEffect, useCallback, SetStateAction } from 'react';
+import PropTypes from 'prop-types';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+interface TabTitleProps extends HTMLElementProps<'button'> {
+  active: boolean;
+  className?: string;
+  id?: string;
+  children: Array<React.ReactElement>;
+  disabled?: boolean;
+  dataTabId: string;
+  onClick?: React.MouseEventHandler;
+  onKeyDown?: React.KeyboardEventHandler;
+  ariaControls?: string;
+  setFocusedState: React.Dispatch<SetStateAction<Array<string>>>;
+  as: React.ElementType<any>;
+}
+
+function TabTitle({
+  active,
+  children,
+  disabled,
+  dataTabId,
+  onClick,
+  onKeyDown,
+  ariaControls,
```

Comment:
What's the reason for explicitly declaring this?

Reply:
https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-controls
From the W3C docs, a use case of aria-controls: "A tab controls the display of its associated tab panel."
Aria-controls denote a relationship, or when the presence of one thing is controlled by something else